gulpfile.js

```diff
@@ -0,0 +1,62 @@
+
+/*
+ * This file is part of the Lug package.
+ *
+ * (c) Eric GELOEN <geloen.eric@gmail.com>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+var gulp = require('gulp');
+var concat = require('gulp-concat');
+var gulpif = require('gulp-if');
+var uglify = require('gulp-uglify');
+var uglifycss = require('gulp-uglifycss');
+
+var prod = process.env.GULP_ENV === 'prod';
+var dest = 'web/assets/';
+
+var paths = {
+    js: [
+        'node_modules/jquery/dist/jquery.min.js',
+        'node_modules/bootstrap/dist/js/bootstrap.min.js',
+        'node_modules/admin-lte/dist/js/app.min.js',
+        'src/Bundle/**/Resources/js/**/*.js' FIXME
+    ],
+    css: [
+        'node_modules/bootstrap/dist/css/bootstrap.min.css',
+        'node_modules/admin-lte/dist/css/AdminLTE.min.css',
+        'node_modules/admin-lte/dist/css/skins/skin-blue.min.css',
+        'node_modules/font-awesome/css/font-awesome.min.css'
+    ],
+    fonts: [
+        'node_modules/font-awesome/fonts/**'
+    ]
+};
+
+gulp.task('js', function () {
```

Comment:
Thanks a lot for the feedback. I was not aware of this gulp module but as the current script is pretty light and the tasks are too, I would prefer keep everything in a single file and if tomorrow, it becomes too fat, I will split them using `wrench`.